src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
```

Comment:
I see. It seems you guys have been using tty-3 for RC and eagle_default for testing.  Right?
I will check if we can change configuration to use tty-3.